tests/sqllogictests/suites/query/02_function/02_0048_function_semi_structureds_parse_json

```diff
@@ -101,7 +101,7 @@ select try_to_variant(123)
 query T
 select try_to_variant(12.34)
 ----
-12.34
```

Comment:
Why does this change the behavior of variant?

Reply:
decimal in json is encoded as string to preserve precision.

Reply:
That's a little unreasonable, @b41sh can we have decimals in jsonb ?

Reply:
maybe we can sacrifice the preserve of precision for json/variant for now?
all other things seems to be ok now

Reply:
I`ll show it another pr.

Reply:
https://github.com/datafuselabs/databend/pull/10346

Reply:
fixed